cmd/add.go

```diff
@@ -23,8 +23,8 @@ func AddCommand() *cobra.Command {
 		newAddSubnetValidatorCommand(),
 	)
 	cmd.PersistentFlags().StringVar(&publicURI, "public-uri", "https://api.avax-test.network", "URI for avalanche network endpoints")
-	cmd.PersistentFlags().StringVar(&privKeyPath, "private-key-path", ".subnet-cli.pk", "private key file path")
-	cmd.PersistentFlags().BoolVarP(&useLedger, "ledger", "l", false, "use ledger to sign transactions")
+	cmd.PersistentFlags().StringVar(&privKeyPath, "private-key-path", ".subnet-cli.pk", "hexadecimal-encoded private key file (either must be set between '--private-key-path' or '--ledger' but not both)")
+	cmd.PersistentFlags().BoolVarP(&useLedger, "ledger", "l", false, "use ledger to sign transactions (either must be set between '--private-key-path' or '--ledger' but not both)")
```

Comment:
Can we switch to using ledger by default and use a private key file if it's set to a non-empty string?